tests/by-util/test_ps.rs

```diff
@@ -0,0 +1,25 @@
+// This file is part of the uutils procps package.
+//
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+// spell-checker:ignore (words) symdir somefakedir
+
+use crate::common::util::TestScenario;
+
+#[test]
+fn test_invalid_arg() {
+    new_ucmd!().arg("--definitely-invalid").fails().code_is(1);
+}
+
+#[test]
+#[cfg(target_os = "linux")]
```

Comment:
please add more tests, esp on other operating systems (at least unix) :)

Reply:
Hmmm ...... It looks like it's very difficult to implement the `ps` command on UNIX platforms as portable, as it's only portable if you have a `proc` pseudo-fs. In other words, the current design only even supports Linux.

- FreeBSD's `proc` pseudo-fs is not automounted.
- macOS does not have a `proc` fs, it use sysctl instead.

Reply:
If we need to do support, we need to refactor uucore's `proc_info` in the future, so that instead of reading the /proc dummy filesystem directly, we can use the syscalls directly.